_data/transport.yml

```diff
@@ -168,6 +168,12 @@ websites:
     facebook: ScotRail
     img: scotrail.png
 
+  - name: Sixt
+    url: https://www.sixt.com/
+    twitter: SixtUSA
+    facebook: sixt.autovermietung
```

Comment:
```suggestion
    facebook: sixt
```

Reply:
Thanks for the suggestion! Facebook only showed me the German version due to my location